web3/auto/infura/mainnet.py

```diff
@@ -0,0 +1,12 @@
+from web3 import Web3
+from web3.providers.auto import (
+    load_provider_from_uri,
+)
+
+from .endpoints import (
+    INFURA_MAINNET_DOMAIN,
+    build_infura_url,
+)
+
+_infura_url = build_infura_url(INFURA_MAINNET_DOMAIN)
+w3 = Web3(load_provider_from_uri(_infura_url))
```

Comment:
I know it shouldn't bother me at all, but the other networks have an extra newline before the `w3` and the pattern match fail is bugging me :P. You seriously don't have to change it, unless it starts to bother you too after looking at it. :)